examples/3d/bloom.rs

```diff
@@ -0,0 +1,81 @@
+use bevy::{pbr::BloomSettings, prelude::*};
+use std::{
+    collections::hash_map::DefaultHasher,
+    hash::{Hash, Hasher},
+};
+
+fn main() {
+    App::new()
+        .insert_resource(Msaa { samples: 1 }) // 1. MSAA must be turned off
```

Comment:
Why?

Reply:
This is no longer true. I fixed it a few commits up. We can remove this line.